package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@boughtbymany/eslint-config-bbm",
-  "version": "2.2.1",
+  "version": "2.2.2",
```

This can be a minor version change as it's adding a new feature.

```suggestion
  "version": "2.3.0",
```